Extension doesn't work after install

Users on Linux and MacOS are reporting that the extension doesn't work after install:

### Steps to reproduce

1. Install from marketplace
2. Open Command Palette
3. Pick "Syntax Tree: Start"
4. Get error popup

Linux
![image](https://user-images.githubusercontent.com/1385470/179828431-f43ebb47-8f88-4d0e-88b2-420650c65b44.png)

Mac
![image](https://user-images.githubusercontent.com/1385470/179828463-3802205a-ce30-4e6d-af29-35434140aa73.png)



VSCode DevTools shows:


Linux
```
mainThreadExtensionService.ts:111 Activating extension 'ruby-syntax-tree.vscode-syntax-tree' failed: Cannot find module 'semver/functions/parse'
Require stack:
- /home/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/node_modules/vscode-languageclient/lib/node/main.js
- /home/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/node_modules/vscode-languageclient/node.js
- /home/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/out/extension.js
- /opt/visual-studio-code/resources/app/out/vs/loader.js
- /opt/visual-studio-code/resources/app/out/bootstrap-amd.js
- /opt/visual-studio-code/resources/app/out/bootstrap-fork.js.
$onExtensionActivationError @ mainThreadExtensionService.ts:111
```

Mac
```
Activating extension 'ruby-syntax-tree.vscode-syntax-tree' failed: Cannot find module 'semver/functions/parse'
Require stack:
- /Users/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/node_modules/vscode-languageclient/lib/node/main.js
- /Users/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/node_modules/vscode-languageclient/node.js
- /Users/user/.vscode/extensions/ruby-syntax-tree.vscode-syntax-tree-0.3.0/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js.
```

--- a/package.json
+++ b/package.json
@@ -82,8 +82,8 @@
   },
   "scripts": {
     "compile": "tsc -p ./",
-    "package": "vsce package --yarn --githubBranch main",
-    "publish": "vsce publish --yarn --githubBranch main",
+    "package": "vsce package --no-yarn --githubBranch main",
+    "publish": "vsce publish --no-yarn --githubBranch main",
     "test": "node ./out/test/runTest.js",
     "vscode:prepublish": "yarn compile",
     "watch": "tsc --watch -p ./"

